app/views/steps/international/jurisdiction/edit.html.erb

```diff
@@ -1,23 +1,18 @@
 <% title t('.page_title') %>
+<%= step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :international_jurisdiction, legend: { tag: 'span', size: 'm' } do %>
+        <p class="govuk-body-l"><%=t '.lead_text' %></p>
 
-    <p><%=t '.info_html' %></p>
-
-    <%= step_form @form_object do |f| %>
-      <%=
-        f.radio_button_fieldset :international_jurisdiction, legend_options: { class: 'visually-hidden', text: t('.heading') }, inline: true do |fieldset|
-          fieldset.radio_input(GenericYesNo::YES, panel_id: :international_jurisdiction_panel)
-          fieldset.radio_input(GenericYesNo::NO)
-          fieldset.revealing_panel(:international_jurisdiction_panel) do |panel|
-            panel.text_area :international_jurisdiction_details, size: '40x4', class: 'form-control-3-4'
-          end
-        end
-      %>
+        <%= f.govuk_radio_button :international_jurisdiction, GenericYesNo::YES, link_errors: true do %>
+          <%= f.govuk_text_area :international_jurisdiction_details, label: { size: 's' } %>
```

Comment:
Also we can get rid of the `label: { size: 's' }` as the default will do fine in this case, size small and no bold.